### ISSUE TITLE
[CLOSED] Erroneous results of Export URN:NBN process - duplicate records

### ISSUE THREAD

**Issue by rzeh4n** [rzeh4n/czidlo-issue-test#29]
_Issue by **[GoogleCodeExporter](https://github.com/GoogleCodeExporter)** from Monday Jun 15, 2015 at 19:26 GMT_
_Originally opened as https://github.com/rzeh4n/czidlo/issues/29_

----

```
CSV export reportedly contains some records multiple times. For example 
urn:nbn:cz:mzk-0000vc on resolver-test.nkp.cz and process with id 32.
```

Original issue reported on code.google.com by `Martin.R...@gmail.com` on 25 Nov 2013 at 7:41

**Comment by rzeh4n** [comment]
_Comment by **[GoogleCodeExporter](https://github.com/GoogleCodeExporter)** from Monday Jun 15, 2015 at 19:26 GMT_

----

```
Z emailove komunikace:
V jednom výpisu u URN urn:nbn:cz:aee001-00009x nebyla k dispozici data ve 
výpisu. I když v resolveru byla. Napadá Vás důvod? Další příkald 
urn:nbn:cz:mzk-0000vc.

A odpvoěď na bod 6. Částečně jsem už problém rozkryl. Jednalo se o 
proces 32 (ve výpisu řádek 613). Záznam urn:nbn:cz:mzk-0000vc. V původním 
výpisu bylo vše, ale na rozdíl od ostatních nebyl v jednom řádku, ale v 
několika. Proto při rozdělení na sloupce část dat byla mimo. Čím se 
problém trochu modifikuje, jak to, že se záznam zapsal do několika polí v 
CSV a ne jen do jednoho?
```

Original comment by `Martin.R...@gmail.com` on 25 Nov 2013 at 7:45

**Comment by rzeh4n** [comment]
_Comment by **[GoogleCodeExporter](https://github.com/GoogleCodeExporter)** from Monday Jun 15, 2015 at 19:26 GMT_

----

```
Issue 28 has been merged into this issue.
```

Original comment by `Martin.R...@gmail.com` on 25 Nov 2013 at 7:46

**Comment by rzeh4n** [comment]
_Comment by **[GoogleCodeExporter](https://github.com/GoogleCodeExporter)** from Monday Jun 15, 2015 at 19:26 GMT_

----

```
1. dulicate record for urn:nbn:cz:mzk-0000vc:
I examined refered CSV and found string "urn:nbn:cz:mzk-0000vc" only once in 
this line:
"urn:nbn:cz:mzk-0000vc","MONOGRAPH","Chtěl jsem vám dát svobodu ; z ruského 
originálu ... přeložil Jaroslav Piskáček ; ilustroval Zdeněk 
Mlčoch","true","","2013-02-25T15:31:06.620+01:00","","false","false","false"

Missing data for urn:nbn:cz:aee001-00009x:
It's true that urn:nbn:cz:aee001-00009x is missing in CSV produced by process 
32. And it shouldn't be there since "aee001" is not among selected registrars 
"mzk,nkp". All parameters of this process are these:
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] registered: 
2011-11-12T13:24:06.000+01:00 - 2013-11-12T13:24:06.000+01:00
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] registrars: 
mzk,nk
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] intelectual 
entity types: MONOGRAPH,MONOGRAPH_VOLUME,PERIODICAL,PERIODICAL_VOLUME
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] limit to 
records missing cCNB: false
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] limit to 
records missing ISSN: true
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] limit to 
records missing ISBN: true
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] return active 
records: true
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] return 
deactivated records: true
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] include number 
of digital instances: false
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] limit to 
records missing ISSN: true
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] limit to 
records missing ISBN: true
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] return active 
records: true
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] return 
deactivated records: true
2013-11-12 13:20:26,557 INFO  [REGISTRARS_URN_NBN_CSV_EXPORT:32] include number 
of digital instances: false
```

Original comment by `Martin.R...@gmail.com` on 25 Nov 2013 at 7:57
* Changed state: **Invalid**
